subgraph/src/fleek-nfa.ts

```diff
@@ -132,20 +127,105 @@ export function handleCollectionRoleRevoked(
   entity.role = event.params.role;
   entity.toAddress = event.params.toAddress;
   entity.byAddress = event.params.byAddress;
-
   entity.blockNumber = event.block.number;
   entity.blockTimestamp = event.block.timestamp;
   entity.transactionHash = event.transaction.hash;
-
   entity.save();
-
   if (event.params.role === 0) {
     // Role 0 => Owner [Probably going to change this after the ACL refactor.]
     // Should remove the CollectionOwner entity.
-
     store.remove('CollectionOwner', event.params.toAddress.toHexString());
   }
 }
+export function handleTokenRoleGranted(event: TokenRoleGrantedEvent): void {
```

Comment:
Why don't we delete this because we will have it in the git history if we really need it right?

Reply:
Yes, we should.